assets/scripts/streets/StreetName.jsx

```diff
@@ -44,10 +45,61 @@ class StreetName extends React.PureComponent {
     super(props)
 
     this.state = {
-      isHovered: false
+      isHovered: false,
+      modify: false,
+      inpVal: ''
```

Comment:
For clarity, rename `inpVal` here and throughout this component to use a less abbreviated form.